app/models/customer.rb

```diff
@@ -1,3 +1,11 @@
 class Customer < ApplicationRecord
   has_many :invoices
+
+  def name
+    "#{first_name} #{last_name}"
+  end
```

Comment:
I think this might fall under presentation logic and might be better to do in the view or associated helpers.